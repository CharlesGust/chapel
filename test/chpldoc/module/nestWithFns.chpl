/* This module has a comment, but uncommented first method */
module Outer {

  proc outerFn() {

  }

  /* The second method is commented */
  proc outerCommented() {

  }

  module Inner {
    /* This module has no comment, but it's inner function does! */
    proc innerFn() {

    }
  }
}