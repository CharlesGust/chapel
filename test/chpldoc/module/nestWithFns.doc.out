Module: Outer
    This module has a comment, but uncommented first method 
   outerFn()

   outerCommented()
       The second method is commented 

   Module: Outer.Inner
      innerFn()
          This module has no comment, but it's inner function does! 

